comit/src/asset/ethereum/erc20.rs

```diff
@@ -22,11 +27,18 @@ impl Erc20Quantity {
         self.0.to_str_radix(10)
     }
 
-    pub fn from_wei_dec_str(str: &str) -> Result<Self, Error> {
+    pub fn try_from_wei_dec_str(str: &str) -> Result<Self, Error> {
```

Comment:
```suggestion
    pub fn try_from_wei_dec_str(wei: &str) -> Result<Self, Error> {
```
Just to make it clearer as the type is already expressed in the signature.

Reply:
Better, thanks.